pages/project-onboarding.tsx

```diff
@@ -311,7 +311,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     where: {
       userId_type: {
         userId: token.sub as string,
-        type: SpaceType.WorkbookUpload,
+        type: SpaceType.ProjectOnboarding,
```

Comment:
What happens here after we roll this out with the existing database values that are the old values?